packages/web/src/scss/components/CheckboxField/_CheckboxField.scss

```
@@ -52,6 +52,11 @@
     @include form-fields-tools.message();
 }
 
+.CheckboxField--danger .CheckboxField__message {
+    color: form-fields-theme.$message-color-danger;
+}
+
+// Deprecated (#DS-590)
 .CheckboxField--error .CheckboxField__message {
```

```suggestion
.CheckboxField--danger .CheckboxField__message,
// Deprecated (#DS-590)
.CheckboxField--error .CheckboxField__message {
    color: form-fields-theme.$message-color-danger;
}
```

How about this? We can remove the `$message-color-error` variable completely right now, cannot we?

---

I've already set up the deprecated message, but I'll admit I'm afraid of removing it. I would probably leave it for the next major version.

---

Don't be afraid. If it builds it is ok to remove it :)

---

I don't like it when there is a mention of deprecation in one case and I will delete it immediately. I would prefer to deal with it only in the step where we will get rid of old "error" occurrences. If there is a class that takes a value that is also called "error", I don't want to bind it to another value.